### ISSUE TITLE
Illegal invocation error when trying to login

### ISSUE THREAD

**Issue by lorro** [icastillejogomez/degiro-api#7]
Hi,

I just installed your package and am trying to figure out how it works. I've just started with the basic, trying to login but I'm already stuck there :)

```js
import DeGiro from 'degiro-api'

const degiro = new DeGiro({
  username: 'username',
  pwd: 'xxxxx'
})

degiro.login()
  .then((accountData) => console.log('Log in success\n', accountData))
  .catch(console.error)
```

gives me the following error in chrome console
```
TypeError: Failed to execute 'fetch' on 'Window': Illegal invocation
    at Object.default (<anonymous>:1:876)
    at eval (login.js?0ff3:36)
    at new Promise (<anonymous>)
    at Object.loginRequest (login.js?0ff3:15)
    at eval (DeGiro.js?1701:59)
    at new Promise (<anonymous>)
    at DeGiro.login (DeGiro.js?1701:58)
    at VueComponent.created (App.vue?234e:19)
    at invokeWithErrorHandling (vue.runtime.esm.js?2b0e:1854)
    at callHook (vue.runtime.esm.js?2b0e:4219)
```

Any idea what I could be doing wrong here?

Kind regards

**Comment by icastillejogomez** [comment]
This looks like the fetch dependency we're using in the lib are not ready for run in the browser. I'm going to fix that ASAP. 

BTW, if you know about it and want to develop the fix and create a PR feel free!

**Comment by lorro** [comment]
I'm afraid I have limited knowledge to fix this myself :) 

P.S. Just out of curiosity, how should the package be used until right now if it's not ready to run in a browser?

**Comment by icastillejogomez** [comment]
With the terminal. Running with Node.js or Deno

**Comment by icastillejogomez** [comment]
The issue is solved in versión `0.6.3`. 

You can follow this repo with a demo in brower: https://github.com/icastillejogomez/degiro-api-web-test
